Update equipment.md

josephderosales opened the pull request:
Added section to cover BYOD

josephderosales committed:
Update equipment.md

Added section to cover BYOD

Commit:
Format code

afeld reviewed (changes requested):
Mind confirming the final language with whomever you were speaking with? Also, this isn't (or shouldn't be) specific to TTS, so right after we merge, mind kicking off an email asking for the content to be moved to InSite?

josephderosales committed:
Update _pages/getting-started/equipment.md

Co-authored-by: Aidan Feldman <aidan.feldman@gsa.gov>

josephderosales committed:
Update equipment.md

updated to include ispcompliance@gsa.gov, reworded and formatted some items as well

Commit:
Format code

afeld reviewed (approved):
Something got messed up with this branch; rebased, and hopefully didn't lost anything. @josephderosales Look ok?

josephderosales committed:
Update _pages/getting-started/equipment.md

Co-authored-by: Aidan Feldman <aidan.feldman@gsa.gov>

afeld committed:
spelling fix
